tests/components/ActionButton/ToggleableButton.spec.ts

```diff
@@ -0,0 +1,27 @@
+describe('ToggleUnactivatedState', () => {
```

Comment:
You can remove the first describe block. Its test is also present in the second block.